packages/design-system/src/components/Banner/Banner.js

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import getMergedClassNames from '../../utils/getMergedClassNames';
+import CloseIcon from 'react-material-icon-svg/dist/CloseIcon';
+import AlertIcon from 'react-material-icon-svg/dist/AlertIcon';
+import InformationOutlineIcon from 'react-material-icon-svg/dist/InformationOutlineIcon';
+import BlockHelperIcon from 'react-material-icon-svg/dist/BlockHelperIcon';
+import CheckCircleIcon from 'react-material-icon-svg/dist/CheckCircleIcon';
+
+import styles from './style.scss';
+
+const cx = classNames.bind(styles);
+
+const baseClass = 'banner';
+
+const getIconByType = (type) => {
+  switch(type) {
+    case 'info':
+      return <InformationOutlineIcon fill="#4384f5" width="20px" height="20px"/>;
+    case 'warning':
+      return <AlertIcon fill="#fb9d01" width="20px" height="20px"/>;
+    case 'success':
+      return <CheckCircleIcon fill="#38c776" width="20px" height="20px"/>;
+    case 'error':
+      return <BlockHelperIcon fill="#d64646" width="20px" height="20px"/>;
+    default:
+      return <AlertIcon fill="#fb9d01" width="20px" height="20px"/>;
```

Comment:
Why this is default? Also if warning and default is the same component, it would be easier to read when you place warning case above default, like:

```javascript
case 'warning':
default:
  return <AlertIcon fill="#fb9d01" width="20px" height="20px"/>;
```